docs/source/contribute.rst

```diff
@@ -2,12 +2,64 @@
 Contribute
 ##########
 
-Vircadia is an open-source project, so all of our code and documentation is available for you to look at. You too can contribute to our endeavor for the metaverse! 
+Vircadia is an open-source project, so all of our code and documentation is available for you to look at. You too can contribute to our endeavor for the metaverse!
+Our main `GitHub repository <https://github.com/kasenvr/project-athena>`_ contains the source to many of the components of our software for creating virtual worlds.
```

Comment:
```suggestion
Our main `git repository <https://github.com/kasenvr/project-athena>`_ contains the source to many of the components of our software for creating virtual worlds.
```

Reply:
The reason for this is to keep agnostic what service is hosting the overall git repos as that can change from GitHub at anytime. Whereas things like "Issues" are specific to GitHub potentially and the wording may have to change if it changes from GitHub anyway.